mappings/net/minecraft/item/FlintAndSteelItem.mapping

```diff
@@ -1 +1,7 @@
 CLASS bbq net/minecraft/item/FlintAndSteelItem
+	METHOD a isLightableBlock (Lbvj;)Z
```

Comment:
maybe `isIgnitableBlock`?

Reply:
The method returns whether it's a block that can be lit (currently only the campfire). I think "lightable" describes this purpose (you *light* a campfire, not *ignite* it, and you *set* wood *on fire*, not *light* it).